cmake/custom/FindPythonLibsNew.cmake

```diff
@@ -1,195 +0,0 @@
-# - Find python libraries
```

Comment:
This file should be kept. It comes from pybind11 and it's the only CMake `Find`-package (that i know of) that will find a consistent version of interpreter and libraries on your machine. The stock `FindPythonLIbs.cmake` can (and will, given the chance!) get you a library that has nothing to do with your interpreter. Please restore this file and use `find_package(PythonLibsNew 2.7 REQUIRED)` instead of the `find_package(PythonInterp 2.7 REQUIRED)` and `find_package(PythonLibs 2.7 REQUIRED)`